src/wallet/walletdb.h

```diff
@@ -41,6 +41,24 @@ enum DBErrors
     DB_NEED_REWRITE
 };
 
+class CWalletScanState {
```

Comment:
I'd say this belongs in wallet.h, not walletdb.h, as it's not used in walletdb anymore?